packages/ibm-products/src/components/APIKeyModal/APIKeyModal.docs-page.js

```diff
@@ -7,23 +7,23 @@
 
 import React from 'react';
 import { StoryDocsPage } from '../../global/js/utils/StoryDocsPage';
-import { useOf } from '@storybook/blocks';
-import { storyDocsGuidelines } from '../../global/js/utils/story-helper';
+
 import * as stories from './APIKeyModal.stories';
 
 const DocsPage = () => {
-  const { csfFile } = useOf('meta', ['meta']);
-
   return (
     <StoryDocsPage
       altGuidelinesHref={[
-        storyDocsGuidelines(csfFile),
+        {
+          href: 'https://pages.github.ibm.com/carbon/ibm-products/components/api-key-modal/usage/',
+          label: 'APIKeyModal usage guidelines',
+        },
         {
           href: 'https://www.carbondesignsystem.com/components/modal/usage',
           label: 'Carbon Modal usage guidelines',
         },
         {
-          href: 'https://react.carbondesignsystem.com/?path=/docs/modal--default',
+          href: 'https://react.carbondesignsystem.com/?path=/docs/components-modal--overview',
```

Comment:
```suggestion
          href: 'https://react.carbondesignsystem.com/?path=/docs/components-modal',
```

While this updates correctly, I think removing `--overview` is actually safer in case there are future changes to their story organization.

Reply:
Ah. I didn't know that was a thing.